src/client/render/Scene.cpp

```diff
@@ -187,6 +180,42 @@ namespace render {
             i++;
         }
 
+
+        //Capacity button
+        
+        state::Player currentPlayer = listOfPlayerOrder[0];
+        bool found=false;
+
+        for(const auto& button: listOfButtons){
+            if(button.name==capacity){
+                found=true;
```

Comment:
A réfactore, faire un for each pour juste ça est un peu lourd
Utiliser les sous état

Reply:
Je ne vois pas ce que tu veux dire. Je ne vois pas le rapport avec les sous états ici